io.github.Arian94.Tiny-beast-dictionary.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>io.github.Arian94.Tiny-beast-dictionary</id>
+  <developer_name>Arian Pourarian</developer_name>
+  <launchable type="desktop-id">io.github.Arian94.Tiny-beast-dictionary.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GFDL-1.3-or-later</project_license>
+  <name>Tiny Beast Dictionary</name>
+  <summary>Tiny version of Google Translate capable of using huge offline dictionaries</summary>
+  <description>
+  <strong> **Tiny Beast is a free tiny-but-powerful app (< 9 MB) which is used to translate words.**</strong>
+  <p>
+    Online Mode: It gathers translations and examples from trusted resources such as Google and AI services.
+  </p>
+  <p>
+    Offline Mode: For those who want to have dictionaries in their computers locally without having to pay, there are several huge dictionaries (thanks to kaikki.org) dedicated to popular languages in which detailed information such as root of words (etymology), variations in different countries etc. are provided.
+  </p>
+  </description>
+  <releases>
+    <release version="1.9.43" date="2023-03-25"/>
+  </releases>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://github.com/Arian94/flathub/tree/tiny-beast-dictionary/1.png</image>
+      <image type="source">https://github.com/Arian94/flathub/tree/tiny-beast-dictionary/2.png</image>
```

Comment:
these URL are not the file, but the web page in the github UI. so it won't work.